modules/api/src/main/java/io/fluo/api/config/FluoConfiguration.java

```diff
@@ -269,6 +280,39 @@ public FluoConfiguration setWorkerThreads(int numThreads) {
   public int getWorkerThreads() {
     return getInt(WORKER_NUM_THREADS_PROP, WORKER_NUM_THREADS_DEFAULT);
   }
+  
+  public ObserverConfiguration getObserverConfig(String key) {
```

Comment:
could make this mirror the set method and have `List<ObserverConfiguration> getObservers()` instead